src/pdm/cli/utils.py

```diff
@@ -156,6 +157,36 @@ def parse_known_args(self, args: Any = None, namespace: Any = None) -> Any:
         return args, argv
 
 
+def find_similar_text(origin_name: str, target_names: list[str], ratio: float = 0.5) -> list[str]:
+    results = []
+    for name in target_names:
+        ratio = SequenceMatcher(None, origin_name, name).ratio()
+        if ratio > 0.4:
+            results.append((name, ratio))
+    return [name for name, _ in sorted(results, key=lambda x: x[1], reverse=True)]
+
+
+class ArgumentParserSimilarComandUtil:
```

Comment:
Just a personal flavor thing, I don't like static-method-only classes. Especially when the color attribute is removed, we can just make `format_similar_command` into a plain function.

Reply:
LGTM